### PR TITLE
Menu configuration

### PR DESCRIPTION
**Pull request by EnigmaCurry**
This refactors the Traefik `make config` to use a menu driven system made with script-wizard.

https://asciinema.org/a/tGVn2mc8mPNPKIk5wKKHCyjxX

### CODE REVIEW

**Comment by EnigmaCurry** [comment]
I've added a menu to traefik's `make config`.
I've also added a new root command: `make install` which provides a menu to quickly install any project. You can also access it via `d.rymcg.tech install`.

File: _scripts/install [hunk 1]
```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+BIN=$(dirname ${BASH_SOURCE})
+source ${BIN}/funcs.sh
+set -e
+
+check_var ENV_FILE ROOT_ENV ROOT_DIR
+
+${BIN}/confirm yes "This tool is provided for your convenience, please read the README of each project before installation."
```
Comment (mcmikemn) [inline _scripts/install:9]: Without providing a `default` argument, `_scripts/confirm` adds a period after your question; so having a period after your question results in 2 periods. 

Remove the period:
`${BIN}/confirm yes "This tool is provided for your convenience, please read the README of each project before installation"`

**Comment by mcmikemn** [comment]
Oh, and the handling of unsynced vars between `.env-dist` and `.env_{context}_{instance}` works great!

**Comment by EnigmaCurry** [comment]
Thanks for the suggestions, I implemented them all I think.

**Review by mcmikemn — COMMENTED** [review]
Thanks for the changes!

In `d.rymcg.tech/make install` there is no menu item to exit the wizard, and if I press `<esc>` or `<ctrl>-c`, it exits but with an error: `make: *** [Makefile:125: install] Error 1`.

**Comment by EnigmaCurry** [comment]
I've upgraded it to script-wizard 0.1.27 and improved the Ctrl-C handling, needs new testing.

**Comment by EnigmaCurry** [comment]
This should be fixed going forward if you delete _scripts/script-wizard this one time.

**Comment by mcmikemn** [comment]
I deleted `_scripts/script-wizard` and ran `make config`:
```
$ make config
./_scripts/install_script-wizard
Looking for jq ... found /usr/bin/jq
Found supported OS: Linux
Found supported architecture: x86_64

This utility can automatically install a required helper tool called script-wizard.
See https://github.com/enigmacurry/script-wizard

? Do you wish to automatically install script-wizard into `_scripts/script-wizard`? (Y/n):
## Running script-wizard installer ...
## See https://github.com/enigmacurry/script-wizard
Looking for curl ... found /usr/bin/curl
## Downloading ...
+ curl -LO https://github.com/Enigmacurry/script-wizard/releases/download/v0.1.27/script-wizard-Linux-x86_64.tar.gz
% Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
Dload  Upload   Total   Spent    Left  Speed
100     9  100     9    0     0     25      0 --:--:-- --:--:-- --:--:--    25

gzip: stdin: not in gzip format
tar: Child returned status 1
tar: Error is not recoverable: exiting now
make: *** [_scripts/Makefile.globals:50: script-wizard] Error 2
```

**Comment by EnigmaCurry** [comment]
Thats because I failed to push the tag. Its building now https://github.com/EnigmaCurry/script-wizard/actions/runs/6814940129

**Comment by mcmikemn** [comment]
I DLed the newest script-wizard, but `d.rymcg.tech/make install` still exists with error on `<esc>` or `<ctrl>-c`.

This isn't the end of the world. Do you want to merge this branch and make this exit-error thing a separate issure to [maybe] deal with at some point in the future?

**Comment by EnigmaCurry** [comment]
I made it at least say Cancelled, and then you see the error from Make. Make does not have an option to turn off the error printing as far as I can tell.

**Review by mcmikemn — APPROVED** [review]

**Pull request merged**